ios/hypertrack_plugin.podspec

```diff
@@ -17,7 +17,7 @@ A new flutter plugin project.
   s.source_files        = 'Classes/**/*'
   s.public_header_files = 'Classes/**/*.h'
   s.dependency          'Flutter'
-  s.dependency          'HyperTrack/Objective-C', '4.7.0'
+  s.dependency          'HyperTrack/Objective-C', '4.12.3'
```

Comment:
@AshishSynoriq what tests did you perform on the iOS side of things to confirm the methods work with the updated SDK?

Reply:
I am testing things rn, I'll raise another update today for iOS